_docs/_latest/configure/scanner/descriptions.md

```diff
@@ -166,3 +166,17 @@ than the max age.
 For examples of how to define scanner rules for your service account keys, see the
 [`service_account_key_rules.yaml`](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/rules/service_account_key_rules.yaml)
 file.
+
+
+## External Project Access Scanner
+
+The External Project Access Scanner mitigates data exfiltration by identifying users who have access to projects outside of your organization or folder.
+
+## What makes this scanner different?
```

Comment:
Can you please remove this `##` summary line?  It's a bit consistent with the formating on the rest of this page.

Reply:
changed to internal link

Reply:
removed